build-logic/documentation/src/main/groovy/gradlebuild/docs/GradleUserManualPlugin.java

```diff
@@ -158,7 +159,7 @@ private void generateUserManual(Project project, TaskContainer tasks, ProjectLay
             attributes.put("reproducible", "");
             attributes.put("docinfo", "");
             attributes.put("lang", "en-US");
-            attributes.put("encoding", "utf-8");
```

Comment:
fixed globally as small only 13 changes. Kindly ask if to extract its good candicante. Somehow prone issues warnings imho failing the build.